build-rust-xcode.sh

```diff
@@ -1,11 +1,19 @@
 #!/bin/bash
 
-if [[ -f "${HOME}/.bash_profile" ]]; then
-    source "${HOME}/.bash_profile"
+# When building from Xcode we want to ensure that `cargo` is in PATH.
+# as a convenience, add the default cargo install location
+export PATH="$PATH:${HOME}/.cargo/bin"
+
+# Users can optionally set cargo path in xi-mac/.env
+if [[ -f "${SRCROOT}/.env" ]]; then
+    source "${SRCROOT}/.env"
+    export PATH="$PATH:$CARGO_PATH"
```

Comment:
Perhaps we should only make this change if CARGO_PATH is set? Additionally, it may be better to add it to the start of the path so it's resolved properly if there's another cargo binary in the path.

Reply:
Sounds good. :)